dockerfiles/readme.md

```diff
@@ -0,0 +1,5 @@
+# Dockerfiles for the Azure IoT SDK for Python
+
+The current implementation of the Python SDK is a wrapper over the C SDK and has external dependencies that cannot be installed with pip (Boost, libssl, curl...). The versions for these dependencies have to match exactly (because of how C linkers work).
```

Comment:
>wrapper over the C SDK [](start = 50, length = 22)

"wrapper over the C SDK"  -nit but I'd recommend spelling out "wrapper over the Azure IoT C SDK".